packages/suite/src/utils/wallet/nftUtils.ts

```diff
@@ -0,0 +1,60 @@
+import { isTokenDefinitionKnown, TokenDefinition } from '@suite-common/token-definitions';
+import { isNftMatchesSearch, filterNftTokens } from '@suite-common/wallet-utils';
+import { NetworkSymbol, getNetworkFeatures } from '@suite-common/wallet-config';
+import { Token } from '@trezor/blockchain-link-types/src/blockbook-api';
+
+type GetNfts = {
+    tokens: Token[];
+    symbol: NetworkSymbol;
+    nftDefinitions?: TokenDefinition;
+    searchQuery?: string;
+};
+
+export type NftType = 'ERC721' | 'ERC1155';
+
+export const getNfts = ({ tokens, symbol, nftDefinitions, searchQuery }: GetNfts) => {
```

Comment:
Looks like getTokens, would it be possible to unify it?

Reply:
unified